src/main/kotlin/racingcar/view/response/GameResult.kt

```diff
@@ -1,18 +1,27 @@
 package racingcar.view.response
 
 class GameResult(
-    val repsResults: MutableList<RepsResult> = mutableListOf()
+    val repsResults: MutableList<RepsResult> = mutableListOf(),
+    val winners: MutableList<Winner> = mutableListOf()
```

Comment:
이 두 프로퍼티가 아무래도 public 하게 접근이 가능하고 가변적인 컬렉션이다 보니 외부에서 변경이 가능해보여요.
이러한 경우 아래와 같이 표현해서 가변성을 가리기도 합니다~!

```suggestion
    private val _repsResults: MutableList<RepsResult> = mutableListOf(),
    private val _winners: MutableList<Winner> = mutableListOf(),
) {
    val repsResults: List<RepsResult>
        get() = _repsResults.toList()

    val winners: List<Winner>
        get() = _winners.toList()
```

Reply:
오.. 책에서 봤던 내용이었던것 같아요. 궁금한점이 하나 있는데, List를 리턴하는 get function을 하나 정의하지 않는 이유가 있나요? Class.propertyName으로 접근할 수 있도록 짜기 위함일까요?